C-Sharp/math/Armstrong_Number.cs

```diff
@@ -0,0 +1,39 @@
+// Armstrong Number
+using System;
+public class Armstrong // Class Declaration
+{
+  public static void Main(string[] args) {
+    int number,
```

Comment:
Declare all the variables in one line. 
ex - ```int number, r, sum = 0, temp;```

Reply:
done